Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsActivity.java

```diff
@@ -222,7 +222,10 @@ public void showLoading()
   @Override
   public void hideLoading()
   {
-    dialog.dismiss();
+    if (dialog != null)
```

Comment:
Will there ever be a situation where we need this check?

Reply:
No, I don't think it's actually needed anymore. I added it before I simplified `prepareDolphinDirectoriesIfNeeded` by removing the `areDolphinDirectoriesReady` check.